test/integration/mig_autoscaler/controls/mig_autoscaler.rb

```diff
@@ -0,0 +1,129 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project_id = attribute('project_id')
+credentials_path = attribute('credentials_path')
+region     = attribute('region')
+
+ENV['CLOUDSDK_AUTH_CREDENTIAL_FILE_OVERRIDE'] = File.absolute_path(
+  credentials_path,
+  File.join(__dir__, "../../../fixtures/mig/autoscaler"))
+
+expected_instances = 4
+expected_instance_groups = 1
+
+control "MIG" do
+  title "Autoscaling Configuration"
+
+  describe command("gcloud --project=#{project_id} compute instances list --format=json --filter='name~^mig-autoscaler*'") do
+    its(:exit_status) { should eq 0 }
+    its(:stderr) { should eq '' }
+
+    let!(:data) do
+      if subject.exit_status == 0
+        JSON.parse(subject.stdout)
+      else
+        {}
```

Comment:
It appears that `data` is expected to be an array.
```suggestion
        []
```